modules/i3s/src/lib/parsers/parse-i3s.ts

```diff
@@ -91,7 +91,7 @@ export async function normalizeTilesetData(tileset : SceneLayer3D, options : Loa
     root = await nodePagesTile.formTileFromNodePages(0);
   } else {
     // @ts-expect-error options is not properly typed
-    const rootNodeUrl = getUrlWithToken(`${tileset.url}/nodes/root`, options.i3s?.token);
+    const rootNodeUrl = getUrlWithToken(`${url}/nodes/root`, options.i3s?.token);
```

Comment:
It looks to me that the `ts-expect-error` above hid this error from the compiler. Can that be removed with some extra effort? 

We should make regular efforts to try to remove such comments as they defeat the typing we have worked so hard to put in place.

Reply:
Done